.travis.yml

```diff
@@ -0,0 +1,63 @@
+language: python
+
+cache:
+  pip: true
+  apt: true
+
+sudo: false
+dist: trusty
+
+
+matrix:
+  include:
+    - env: TEST=standard
+      os: linux
+      python: '2.7'
+    - env: TEST=standard
+      os: linux
+      python: '3.6'
+
+
+before_install:
+    # Instal miniconda
+    - if [ "${TRAVIS_OS_NAME}" == "linux" ]; then
+        wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+      else
+        wget http://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh;
```

Comment:
you don't need this right? I don't see `mac` in the matrix above ...

Reply:
yes, I let it just in case. But if we don't want OSx test, this can be removed